.gitignore

```diff
@@ -3,3 +3,4 @@
 test/setup/tmp-disposable-nodes-addrs.json
 dist
 coverage
+.DS_Store
```

Comment:
You can ignore .DS_Store globally. See how here: http://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder :)